### PR TITLE
Update the README with documentation on how to use this

### PR DESCRIPTION
**Pull request by justafish**
This also changes the Drupal tasks to use an explicit variable for the site instead of relying on the CLI_ARGS

### CODE REVIEW

**Review by rabbitlair — APPROVED** [review]
💯

File: README.md [hunk 1]
```diff
@@ -1,25 +1,85 @@
...
 
-## Dependencies
-This project requires [Task](https://github.com/go-task/task), [version 3 or later](https://taskfile.dev/#/taskfile_versions) to be installed.
+Drainpipe is a composer package which provides a number of build tool helpers
+for a Drupal site, including:
 
-## Setup
-An initial `Taskfile.yml` will be created when installing this package. Be sure to commit it to version control. To simplify future updates, consider including additional Taskfiles instead of adding commands to the file directly.
+- Site and database updates
+- Artefact packaging for deployment to a hosting provider
```
Comment (deviantintegral) [inline README.md:7]:
```suggestion
- Artifact packaging for deployment to a hosting provider
```

File: README.md [hunk 2]
```diff
@@ -1,25 +1,85 @@
...
+- Test setups
```
Comment (deviantintegral) [inline README.md:8]:
```suggestion
- Automated testing setup with support for PHPUnit and Nightwatch tests
```

File: README.md [hunk 3]
```diff
@@ -1,25 +1,85 @@
...
 
-## Usage
-Below are some examples based on task files included in this package for certain project types.
+## Installation
 
-### Drupal
 ```
-task drupal:update
+composer require lullabot/drainpipe
+composer require lullabot/drainpipe-dev --dev
```
Comment (deviantintegral) [inline README.md:14]:
```suggestion
# Includes development dependencies, but only in the `require-dev` section. This step is required for Drainpipe to function correctly.
composer require lullabot/drainpipe-dev --dev
```

Unless it's actually optional, in which case we should call it out!

File: README.md [hunk 4]
```diff
@@ -1,25 +1,85 @@
...
 ```
 
-or for a remote site:
+There are a number of pre-configurations provided if you're using
+[DDEV](https://ddev.readthedocs.io/en/stable/), so either set DDEV up first
+before requiring this project, or run `composer update` if DDEV is added later.
```
Comment (deviantintegral) [inline README.md:19]:
```suggestion
Drainpipe integrates with [DDEV](https://ddev.readthedocs.io/en/stable/), but
will only add the relevant files when DDEV is detected in the repository. Either
set DDEV up first before requiring this project, or run `composer update` if
DDEV is added later.
```

File: README.md [hunk 5]
```diff
@@ -1,25 +1,85 @@
...
+cd drupal
+ddev config
+composer require lullabot/drainpipe
+composer require lullabot/drainpipe-dev --dev
+ddev start
 ```
 
+## Usage
+
+Helpers are provided as [Taskfiles](https://taskfile.dev/#/). A full list of
```
Comment (deviantintegral) [inline README.md:32]:
```suggestion
Build tasks are provided as [Taskfiles](https://taskfile.dev/#/). A full list of
```

**Review by deviantintegral — COMMENTED** [review]
A few suggestions! Nice 👏 .

File: README.md (hunk 3 above)
Comment (justafish) [reply README.md:14]: It's not required for it to functional correctly!

File: README.md [hunk 6]
```diff
@@ -1,25 +1,87 @@
...
+DDEV is added later.
 
 ```
-task drupal:update -- @mysite.dev
+composer create-project drupal/recommended-project drupal
+cd drupal
+ddev config
+composer require lullabot/drainpipe
+composer require lullabot/drainpipe-dev --dev
+ddev start
```
Comment (justafish) [inline README.md:29]:
```suggestion
ddev config
ddev start
ddev composer require lullabot/drainpipe
ddev composer require lullabot/drainpipe-dev --dev
# Restart is required to enable the provided Selenium containers
ddev restart
```

**Pull request merged**